group-coordinator/src/main/java/org/apache/kafka/coordinator/group/metrics/GroupCoordinatorRuntimeMetrics.java

```diff
@@ -202,6 +248,19 @@ public void recordEventQueueTime(long durationMs) { }
     @Override
     public void recordEventQueueProcessingTime(long durationMs) { }
```

Comment:
Should we also implement `recordEventQueueTime` and `recordEventQueueProcessingTime`? I think that we need histogram for those too.

Reply:
I was planning to add them in a separate PR. i can add it here if you prefer that

Reply:
It may be better to add them in this one so we can look at all those metrics together.